modules/nf-core/muse/sump/meta.yml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "muse_sump"
+description: computes tier-based cutoffs from a sample-specific error model
+keywords:
+  - variant calling
+  - somatic
+  - wgs
+  - wxs
+  - vcf
+tools:
+  - "MuSE":
+      description: "Somatic point mutation caller based on Markov substitution model
+        for molecular evolution"
+      homepage: "https://bioinformatics.mdanderson.org/public-software/muse/"
+      documentation: "https://github.com/wwylab/MuSE"
+      tool_dev_url: "https://github.com/wwylab/MuSE"
+      doi: "10.1101/gr.278456.123"
+      licence: ["https://github.com/danielfan/MuSE/blob/master/LICENSE"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - muse_call_txt:
+        type: file
+        description: single input file generated by 'MuSE call'
+        pattern: "*.MuSE.txt"
+  - - meta2:
+        type: map
+        description: |
+          Groovy Map containing reference information.
+          e.g. `[ id:'test' ]`
+    - ref_vcf:
+        type: file
+        description: |
+          dbSNP vcf file that should be bgzip compressed, tabix indexed and
+          based on the same reference genome used in 'MuSE call'
+        pattern: ".vcf.gz"
+    - ref_vcf_tbi:
+        type: file
+        description: |
+          dbSNP vcf file that should be bgzip compressed, tabix indexed and
+          based on the same reference genome used in 'MuSE call'
```

Comment:
```suggestion
          Tabix index for the dbSNP vcf file
```

Reply:
I think I would leave this as descriptive as possible?